### PR TITLE
DOMA-3282 add payment order field to payment

### PR DESCRIPTION
**Pull request by dkoviazin**
Add payment order field to Payment schema

### CODE REVIEW

**Review by DanilovMaxim13 — APPROVED** [review]

File: apps/condo/domains/acquiring/schema/Payment.js [hunk 1]
```diff
@@ -186,6 +186,12 @@ const Payment = new GQLListSchema('Payment', {
             defaultValue: PAYMENT_INIT_STATUS,
         },
 
+        order: {
+            schemaDoc: 'Payment order number',
+            type: Text,
```
Comment (AleX83Xpert) [inline apps/condo/domains/acquiring/schema/Payment.js:191]: `Text`? Why not `Integer`?

File: apps/condo/migrations/20220630110118-0141_payment_order_paymenthistoryrecord_order.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// auto generated by kmigrator
```
Comment (AleX83Xpert) [inline apps/condo/migrations/20220630110118-0141_payment_order_paymenthistoryrecord_order.js:1]: Please create the new migration file, cause of my [migration](https://github.com/open-condo-software/condo/pull/1835/files#diff-d88e4b725d67b36fa324ee22d110638e68743f68fcce0358782cd964c40ac1ea), sorry :)

File: apps/condo/domains/acquiring/schema/Payment.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/acquiring/schema/Payment.js:191]: Not sure if payment orders will contain special chars or not - dots, slashes .... And it's not really important to have number field here, as we are not going to make math operations on database layer with this field

File: apps/condo/migrations/20220630110118-0141_payment_order_paymenthistoryrecord_order.js (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/migrations/20220630110118-0141_payment_order_paymenthistoryrecord_order.js]: Done

**Review by AleX83Xpert — APPROVED** [review]

**Pull request merged**